plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-logo.tsx

```diff
@@ -526,6 +526,24 @@ export const SidebarNavigationScreenLogo = ( {
 						mediaItemData={ mediaItemData }
 						isLoading={ isLoading }
 					/>
+					<div className="woocommerce-customize-store__fiverr-cta-group">
+						<strong>
+							{ __( "DON'T HAVE A LOGO YET?", 'woocommerce' ) }
+						</strong>
+						<p>
+							{ __(
+								'Get a custom logo designed by a professional on ',
+								'woocommerce'
+							) }
+							<a
+								href="https://www.fiverr.com/logo-maker/woo?afp=&cxd_token=917527_33214203&show_join=true"
+								target="_blank"
+								rel="noreferrer"
+							>
+								{ __( 'Fiverr.', 'woocommerce' ) }
```

Comment:
The text to be translated is broken into two parts here (the sentence "Get a custom logo designed by a professional on Fiverr." is split). This segmentation can lead to translation issues because different languages have different syntactic structures. I would recommend relying on `interpolateComponents` instead. Example:

https://github.com/woocommerce/woocommerce/blob/af2946aa3431d89d0fcbd0748a12628b39b141fc/plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-typography.tsx#L181

Reply:
Nice catch! I've updated to use `interpolateComponents` in 4675ceaf.

Thanks!